FRC1076-2016-Competition/test/org/usfirst/frc/team1076/test/mock/MockRobot.java

```diff
@@ -41,8 +41,7 @@ public void setLidarSpeed(double speed) {
 	}
 
 	public SensorData getSensorData() {

```

Comment:
The interface should be changed to return an `ISensorData`, and all overriding methods should be `@Override`.